cob_mimic/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ add_action_files(DIRECTORY action FILES
 generate_messages(DEPENDENCIES actionlib_msgs)
 
 catkin_package(
-  CATKIN_DEPENDS actionlib_msgs message_runtime
+  CATKIN_DEPENDS actionlib_msgs message_runtime roslib
```

Comment:
I think that it is out-of-date ( before the package xml 2 migration). This dependency was in my old branch play_sound.